### ISSUE TITLE
`stretchH: 'last'` doesn't have a minimum value 

### ISSUE THREAD

**Issue by maksymblank** [handsontable/handsontable#7679]
### Steps to reproduce
1. Launch handsontable with multiple fields
2. Adjust the screen size until the last column is stretched
3. If you minimize screen even more, the last column will get smaller, but the table won't become horizontally scrollable until the last column is < 0px. This means that for some users the last column may be 5-10px wide, which makes it impossible to type in. (screenshots below)

![image](https://user-images.githubusercontent.com/25963776/113624111-d61ac600-9624-11eb-9d5d-a8e72feadd96.png)

### Possible solution:
Add a config `stretchHMinimumWidth` (number) and replace
`lastColumnWidth >= 0 ? lastColumnWidth : columnWidth;` with `lastColumnWidth >= stretchHMinimumWidth ? lastColumnWidth : columnWidth;` at [line 205 in viewportColumns.js (3rdparty/walkontable)](https://github.com/handsontable/handsontable/blob/master/src/3rdparty/walkontable/src/calculator/viewportColumns.js#L205)

### Your environment
* Handsontable version: 7.4.2
* Browser Name and version: Chrome 89.0.4389.114
* Operating System: MacOS

**Comment by AMBudnik** [comment]
Hi @MaksymBlank 

Can you please add a demo where this issue can be replicable?

I'm only experiencing these two issues https://github.com/handsontable/handsontable/issues/6368, https://github.com/handsontable/handsontable/issues/3925, while working on https://handsontable.com/docs/8.3.2/demo-stretching.html demo.

**Comment by MaksymBlank** [comment]
@AMBudnik if you change the screen width to 334px on [https://handsontable.com/docs/8.3.2/demo-stretching.html](https://handsontable.com/docs/8.3.2/demo-stretching.html), the last column gets stretched to 4px wide, which makes it impossible to edit or select (screenshots below).

![image](https://user-images.githubusercontent.com/25963776/113725147-a917f300-96b8-11eb-975d-1397ea39ff8d.png)
![image](https://user-images.githubusercontent.com/25963776/113725108-a0bfb800-96b8-11eb-89a0-33b516f8513f.png)

**Comment by AMBudnik** [comment]
Thank you for sharing @MaksymBlank 

I have replicated the issue on Chrome 89 / Windows 10. (I've also spotted that other tables are missing half of the last header)

![issue](https://user-images.githubusercontent.com/10757813/113823188-a4b60d80-977e-11eb-90b6-dc86b5e2b531.gif)

It seems that the editor still works well.

![image](https://user-images.githubusercontent.com/10757813/113823302-c7e0bd00-977e-11eb-9253-cb343cf60f6c.png)

Interestingly, when I went back to the version picker and chosen 8.3.1, everything worked. I went back to 8.3.2, and it worked there as well. We should test that locally.

**Comment by AMBudnik** [comment]
Vanilla 8.3.2 on my Chrome shows the same result as shown in the first post (column shows data but it is cut and doesn't provide a scroll)

![image](https://user-images.githubusercontent.com/10757813/113824690-5f92db00-9780-11eb-8211-96a01d6f1cef.png)

**Comment by MaksymBlank** [comment]
@AMBudnik you're right, the editor still works, but it's impossible to read values in the last column (if it's been stretched to 4px)
It would be great if we could set a minimum value for the stretched column 🙌  (e.g. test env: https://github.com/MaksymBlank/handsontable/commit/c2f615bb9d018b7f52f1e6c5d29d4c6bd200f9d7#diff-e53cb26da54338e8f6f8d1cf52ae08c478f120259b480799009271794548b0aaR205)

**Comment by AMBudnik** [comment]
@MaksymBlank Definitely! Now it doesn't work as expected.

I've marked this issues as bug and will update you as soon as we fix it.

**Comment by AMBudnik** [comment]
Hi @maksymblank 

We just released Handsontable v15.0.0 and made a lot of fixes to stretching. It seems that this issue is no longer replicable in the documentation. 

Please update and let me know if it also applies to your application.
